build/textures.rs

```diff
@@ -644,11 +644,16 @@ fn build_texture<W: Write>(mut dest: &mut W, ty: TextureType, dimensions: Textur
 
             TextureDimensions::Texture2dArray => (write!(dest, "
                     let array_size = 0;
```

Comment:
These definitions aren't needed. When you write `let RawImage3d { data, width, etc. } = ...` it creates new variables.